twilio/base/exceptions.py

```diff
@@ -19,12 +19,13 @@ class TwilioRestException(TwilioException):
          not available for all errors.
     """
 
-    def __init__(self, status, uri, msg="", code=None, method='GET'):
+    def __init__(self, status, uri, msg="", code=None, method='GET', details={}):
```

Comment:
Prefer `None` for the default of `details` since it was not present in the response.

Also, class doc above needs added param.

Reply:
@childish-sambino fixed